hedera-mirror-rosetta/cmd/config.go

```diff
@@ -20,11 +19,11 @@ func LoadConfig() *types.Config {
 	var configuration types.Config
 	GetConfig(&configuration, defaultConfigFile)
 	GetConfig(&configuration, mainConfigFile)
-	value, isPresent := os.LookupEnv(config.EnvHederaMirrorRosettaDBHost)
-	if isPresent {
-		log.Println("Config: Database host is set as ENV variable. Will be using that instead of the host specified in application.yml")
-		configuration.Hedera.Mirror.Rosetta.Db.Host = value
+
+	if err := env.Parse(&configuration); err != nil {
+		panic(err)
```

Comment:
I think that it would be meaningful to log some error here as-well so that the dev knows whats going on